### PR TITLE
Feature/orthopairs jenkins and general updates

### PR DESCRIPTION
**Pull request by cookersjs**
Relatively small PR that motivated the creation of a 'compare line counts of files between releases' method that can be seen at the shared library. Aside from that, this PR sees:

- Shift to using shared groovy library
- Added in default values for certain configurations

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -70,20 +64,33 @@ pipeline{
 				}
 			}
 		}
+		// This stage compares the line counts of the orthopairs files generated between the current and previous release.
+		// An intelligible output should be visible at the console logs for the build.
+		stage('Post: Orthopairs file line counts') {
+		    steps {
+		        script {
+		            def releaseVersion = utils.getReleaseVersion()
+		            def previousReleaseVersion = utils.getReleaseVersion().toInteger() - 1
```
Comment (jweiser) [inline Jenkinsfile:73]: Since `previousReleaseVersion` is a common thing we use, maybe a function could for it be added to the shared library?

**Review by jweiser — DISMISSED** [review]
Looks good to me!

**Comment by SolomonShorser-OICR** [comment]
This still depends on an active PR (https://github.com/reactome/release-jenkins-utils/pull/8) so I'm going to wait until that one is finished before approving this one.

File: Jenkinsfile (hunk 1 above)
Comment (cookersjs) [reply Jenkinsfile:73]: Done

Comment (jweiser) [reply Jenkinsfile:72]: Great!  Does the 'getPreviousReleaseVersion()` function call need to be added here?  I don't see that change on my end.

Comment (cookersjs) [reply Jenkinsfile]: Yep! Good catch

**Review by jweiser — APPROVED** [review]

**Pull request merged**